src/website/views/custom-elements/button.pug

```diff
@@ -0,0 +1,268 @@
+---
+title: Button
+scripts:
+- js/ce/ux-chi-ce
+
+---
+p.-text
+  | Buttons are used to trigger actions in forms, modals, and more.
+
+h3 Button
```

Comment:
@srinath-koneru  in CE we are not supporting different tags. In fact, all the buttons render a button, so the doc with link, input... is not necessary. Remove Link, Input, Submit and Reverse from these examples and just leave Button example.